mhubert-pmn/Front-Url-Shortener
# [Style] Configure style

## Task
Configure style

## Type of change

## Description
- Add a reset SCSS file to reset style
- Install node-sass to use SCSS
- Add font for H1
- Add temporary content in the App component

## Testing scenario
In the terminal (you have to be in the front-url-shortener folder):
- Launch and build: `docker-compose up --build`
- List running containers: `docker ps`
- Launch container: `docker exec [container name] /bin/sh`
- Open the app at url `localhost:3000`: you should see a title (using the Lilita One font) and a subtitle
- Down and delete running containers, volumes and networks: `docker-compose down`

## Trello card
[https://trello.com/c/0wxYEx3q](https://trello.com/c/0wxYEx3q)
